src/implementations/twig/components/tag/tag-set.html.twig

```diff
@@ -0,0 +1,48 @@
+{% apply spaceless %}
+
+{#
+  Parameters:
+  - "items" (array) (default: []): array of Tag
+  - "default_icon_path" (string) Path to the icons sprite. If provided here, it will be used for every tag in the set
```

Comment:
we do use icon_path everywhere, we should actually "fix" the datepicker that currently uses icons_path, i wouldn't introduce an inconsistency about this parameter.